Configuration/Settings.yaml

```diff
@@ -12,7 +12,12 @@ Ttree:
   Map:
     apiKey:
       name: "API KEY"
-
+    # Configure GoogleMaps Geocoder
+    geocoder:
+      locale: null
+      region: null
+      useSSL: false
```

Comment:
Maybe true is a good default, no ?

Reply:
Idk, fine for me. But default of geocoder is currently `false`

Reply:
I think, we are in 2018 ;) https should be the default